src/lib/import_export/csv_converter.hpp

```diff
@@ -96,7 +96,17 @@ class CsvConverter : public BaseCsvConverter {
 };
 
 template <>
-inline std::function<int(const std::string&)> CsvConverter<int>::_get_conversion_function() {
+inline std::function<uint8_t(const std::string&)> CsvConverter<uint8_t>::_get_conversion_function() {
+  return [](const std::string& str) {
+    size_t pos;
+    auto converted = std::stoi(str, &pos);
+    Assert(pos == str.size(), "Unprocessed characters found while converting to bool: " + str);
```

Comment:
Converting to bool? I don't see a single bool in this method.

Reply:
Since we don't add the bool type to opossum, we don't need it anymore.